### PR TITLE
Unarchive from command palette

### PR DESCRIPTION
**Pull request by logason**
Docs for how to unarchive from the command palette

### CODE REVIEW

**Comment by hugihlynsson** [comment]
#### 📝 Changed routes:


| [**…ce/archive-and-restore**](https://docs-2opoisc4w.avo-dev.app/workspace/archive-and-restore) |
|:-:|
|<a href="https://docs-2opoisc4w.avo-dev.app/workspace/archive-and-restore">Screenshot of /workspace/archive-and-restore</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/h70_u3CvoPYvmFEPgg8-raL-4Lr2KYfHJ56eaVRuGX_Xevjv03LSn4pYX9jooraEbEGTMBEQgXkZjw_uOUBd2g.png">(view full size)</a> |







Commit 65a95a86a159593b9c8571d0391b3cb5fa65e3f8 (https://docs-2opoisc4w.avo-dev.app).

File: pages/workspace/archive-and-restore.mdx [hunk 1]
```diff
@@ -19,9 +19,32 @@ In Avo you can archive items that you don't want in your tracking plan anymore.
 
 In the top right corner of the item you want to archive, click the `...` menu and select the "Archive ..." option.
 
-### Restoring an item
+### How do I unarchiving an item?
```
Comment (hugihlynsson) [inline pages/workspace/archive-and-restore.mdx:22]:
```suggestion
### How do I unarchive an item?
```

File: pages/workspace/archive-and-restore.mdx [hunk 2]
```diff
@@ -19,9 +19,32 @@ In Avo you can archive items that you don't want in your tracking plan anymore.
...
 
-When an item is archived it is moved from your tracking plan to an archive store within your Avo workspace. Archived items are currently not accessible from the tracking plan UI but can be restored from the Activity log.
+When an item is archived it is moved from your tracking plan to an archive store within your Avo workspace. Archived items can be browsed and unarchived from the Command Palette (cmd+k on Mac, ctrl+k on Windows), in your workspace Activity Log, and on the branch review screen.
```
Comment (hugihlynsson) [inline pages/workspace/archive-and-restore.mdx:24]:
```suggestion
When an item is archived it is moved from your tracking plan to an archive within your Avo workspace. Archived items can be browsed and unarchived from the Command Palette (cmd+k on Mac, ctrl+k on Windows), in your workspace Activity Log and on the branch review screen.
```

**Review by hugihlynsson — APPROVED** [review]
Few nits. Love the video!

**Comment by logason** [comment]
> Few nits. Love the video!

Trying out https://www.screen.studio/ ✨

**Review by thoragudf — APPROVED** [review]
Wowsa nice video! Love it 🚢

**Pull request merged**